frontend/public/locales/en/translation.json

```
@@ -3020,6 +3021,7 @@
   "tooltip.creation.ocp.diskSizeGB": "Disk size (GiB)",
   "tooltip.creation.ocp.gcp.instance.type": "The GCP machine type.",
   "tooltip.creation.ocp.gcp.region": "The GCP region where you deploy your cluster. You can select zones within the region for your control plane and worker pools.",
+  "tooltip.creation.ocp.hosted.cluster.namespace": "The namespace where hosted cluster resources will be created. Do not enter a namespace that matches the cluster's name or use an existing managed cluster namespace.",
```

```suggestion
  "tooltip.creation.ocp.hosted.cluster.namespace": "The namespace where hosted cluster resources are created. Do not enter a namespace that matches the cluster name and do not use an existing managed cluster namespace.",
```

I assume for the last part, it's supposed to be a negative for both?

---

Yup, that's right. Great edits. Thank you!